src/qt/guiutil.cpp

```diff
@@ -613,8 +613,14 @@ bool GetStartOnSystemStartup()
 
 bool SetStartOnSystemStartup(bool fAutoStart)
 {
-    if (!fAutoStart)
-        fs::remove(GetAutostartFilePath());
+    if (!fAutoStart) {
+        try {
+            fs::remove(GetAutostartFilePath());
+        } catch(const fs::filesystem_error& e) {
+            LogPrintf("failed to remove autostart file: %s\n", e.what());
```

Comment:
Could you please Capitalize the first word in the log message to follow other `LogPrintf()` calls in the code base?